src/components/MediaList/MediaList.tsx

```diff
@@ -0,0 +1,296 @@
+  };
+}
+
+const StyledMediaListItemBody = styled.div`
+  ${fonts.size.text.metaText.small};
+  ${mq.range({ from: breakpoints.tablet })} {
+    max-width: 70%;
+  }
+  ${mq.range({ from: breakpoints.desktop })} {
+    max-width: 75%;
+  }
+`;
+
+export const MediaListItemBody = ({
+  children,
+  license: licenseAbbreviation,
+  locale,
+  resourceUrl = "", // defaults to current page
+  resourceType,
+}: MediaListItemBodyProps) => {
+  const license = getLicenseByAbbreviation(licenseAbbreviation, locale);
+  const containerProps = isCreativeCommonsLicense(license.rights)
+    ? {
+        "xmlns:cc": "https://creativecommons.org/ns#",
+        "xmlns:dct": "http://purl.org/dc/terms/",
+        about: resourceUrl,
+      }
+    : {};
+
+  const metaResourceType = getResourceTypeNamespace(resourceType);
+  const hidden = {
+    display: "none",
+  };
+
+  return (
+    <StyledMediaListItemBody {...containerProps}>
+      {/* @ts-ignore */}
+      {metaResourceType && <span rel="dct:type" href={metaResourceType} style={hidden} />}
+      {children}
+    </StyledMediaListItemBody>
+  );
+};
+
+const StyledMediaListItemActions = styled.div`
+  display: flex;
+  align-items: start;
+  margin: ${spacing.small} 0 0 0;
+  width: 100%;
+  button,
+  a {
+    margin: 0 ${spacing.small} ${spacing.small} 0;
+  }
+  ${mq.range({ until: breakpoints.mobileWide })} {
+    display: flex;
+    flex-direction: column;
+    button,
+    a {
+      margin: 0 0 ${spacing.small} 0;
+      width: 100%;
+    }
+    span {
+      width: 100%;
+      button {
+        width: 100%;
+      }
+    }
+  }
+`;
+
+export const MediaListItemActions = ({ children, ...rest }: ComponentProps<"div">) => (
+  <StyledMediaListItemActions {...rest}>{children}</StyledMediaListItemActions>
+);
+
+const isLink = (text: string) => text.startsWith("http") || text.startsWith("https");
+
+interface HandleLinkProps {
+  text: string;
+  children: ReactNode;
+}
+
+export const HandleLink = ({ text, children }: HandleLinkProps) => {
+  if (isLink(text)) {
+    return (
+      <SafeLink to={text} target="_blank" rel="noopener noreferrer">
+        {children}
+      </SafeLink>
+    );
+  }
+  return <span>{children}</span>;
+};
+
+const attributionTypes = [metaTypes.author, metaTypes.copyrightHolder, metaTypes.contributor];
+
+export type ItemType = ItemTypeWithDescription | DescriptionLessItemType;
+
+interface ItemTypeWithDescription {
+  label: string;
+  description: string;
+  metaType: Omit<MetaType, "otherWithoutDescription">;
+}
+
+interface DescriptionLessItemType {
+  label: string;
+  metaType: "otherWithoutDescription";
+}
+
+function isOtherWithoutDescription(item: ItemType): item is DescriptionLessItemType {
+  return item.metaType === metaTypes.otherWithoutDescription;
+}
+
+interface MediaListItemMetaProps {
+  items?: ItemType[];
+}
+
+const ItemText = ({ item }: { item: ItemType }) => {
+  if (isOtherWithoutDescription(item)) {
+    return <>{item.label}</>;
```

Comment:
Du er i typescript-kontekst her så om du berre fjerner fragments og {} så bør det funke.
```suggestion
    return item.label;
```